website/content/v1.9/reference/cli.md

```diff
@@ -209,6 +209,7 @@ talosctl cluster create [flags]
       --with-debug                               enable debug in Talos config to send service logs to the console
       --with-firewall string                     inject firewall rules into the cluster, value is default policy - accept/block (QEMU only)
       --with-init-node                           create the cluster with an init node
+      --with-json-logs                           enable JSON logs sender and receiver
```

Comment:
just receiever only right?

Reply:
both, as it patches Talos itself

Reply:
probably I can reword it to make it more obvious

Reply:
updated wording